pkg/reconcile/reconcile.go

```diff
@@ -17,13 +17,18 @@ limitations under the License.
 package reconcile
 
 import (
+	"time"
+
 	"k8s.io/apimachinery/pkg/types"
 )
 
 // Result contains the result of a Reconciler invocation.
 type Result struct {
 	// Requeue tells the Controller to requeue the reconcile key.  Defaults to false.
 	Requeue bool
+
+	// RequeueAfter is the duration to wait before adding the key back to the queue.
+	RequeueAfter *time.Duration
```

Comment:
We could avoid the pointer in this case by considering zero-value as indicator for not-to-requeue-after. I see that pointer introduces unnecessary verbosity in caller's code by having to declare a variable. WDYT ?

Reply:
Done